request/middleware.py

```diff
@@ -30,7 +33,19 @@ def process_response(self, request, response):
             if request.user.username in settings.REQUEST_IGNORE_USERNAME:
                 return response
 
-        r = Request()
-        r.from_http_request(request, response)
+        req = Request()
+        req.from_http_request(request, response)
+
+        if settings.USE_TRACKING:
+            if 'track_key' in request.COOKIES:
```

Comment:
What about making the key customisable? Perhaps there could be a setting for it.